examples/units_and_coordinates/stereo_loc_sunpy.py

```diff
@@ -0,0 +1,47 @@
+"""
+===================================
+Getting the location of STEREO-A using sunpy
+===================================
+
+How to get the position of planetary bodies im the solar system using
+`astropy's solar system ephemeris <http://docs.astropy.org/en/stable/coordinates/solarsystem.html#solar-system-ephemerides>`__ information and sunpy.
```

Comment:
```suggestion
This example shows how to get and plot the position of planetary bodies within the solar system using
`astropy's solar system ephemeris <http://docs.astropy.org/en/stable/coordinates/solarsystem.html#solar-system-ephemerides>`__ information and sunpy.
```
You can link to the astropy docs without using URLs, see https://coderefinery.github.io/sphinx-lesson/intersphinx.

You should also link to the main sunpy function(s) you are using.